pkger/parser_test.go

```diff
@@ -1012,6 +1012,251 @@ spec:
 			})
 		})
 
+		t.Run("pkg with single dashboard heatmap chart", func(t *testing.T) {
+			testfileRunner(t, "testdata/dashboard_heatmap.json", func(t *testing.T, pkg *Pkg) {
```

Comment:
we need to support both `yml` and `json` for the happy path stuff. Make sure to run this as `testdata/dashboard_heatmap` and have goth yml/json avialble to run